diff --git a/node_modules/@expo/vector-icons/build/createIconSet.js b/node_modules/@expo/vector-icons/build/createIconSet.js
index e8e1ec0..4eea66f 100644
--- a/node_modules/@expo/vector-icons/build/createIconSet.js
+++ b/node_modules/@expo/vector-icons/build/createIconSet.js
@@ -19,6 +19,7 @@ export default function (glyphMap, fontName, expoAssetId, fontStyle) {
         _icon;
         state = {
+            fontIsLoaded: true,
         };
         async componentDidMount() {
            this._mounted = true;
            try {
                await Font.loadAsync({ [fontName]: expoAssetId });
                if (this._mounted) {
                this.setState({ fontIsLoaded: true });
                }
            } catch (e) {
                console.warn(`Failed to load font: ${fontName}`, e);
            }
            }
