diff --git a/node_modules/expo-font/build/Font.js b/node_modules/expo-font/build/Font.js
index 703417f..9470452 100644
--- a/node_modules/expo-font/build/Font.js
+++ b/node_modules/expo-font/build/Font.js
@@ -14,7 +14,7 @@ import { registerStaticFont } from './server';
 export function isLoaded(fontFamily) {
     if (Platform.OS === 'web') {
         if (typeof ExpoFontLoader.isLoaded !== 'function') {
-            throw new Error(`expected ExpoFontLoader.isLoaded to be a function, was ${typeof ExpoFontLoader.isLoaded}`);
+            ExpoFontLoader.isLoaded = () => true;
         }
         return isLoadedInCache(fontFamily) || ExpoFontLoader.isLoaded(fontFamily);
     }
